navigator/backend/BUILD.bazel

```diff
@@ -98,6 +98,7 @@ compileDependencies = [
     "//3rdparty/jvm/org/gnieh:diffson_spray_json",
     "//3rdparty/jvm/org/tpolecat:doobie_core",
     "//3rdparty/jvm/org/xerial:sqlite_jdbc",
+    "//3rdparty/jvm/org/postgresql:postgresql",
```

Comment:
Because I had to create new converters to turn `PGobject`s of `json` type from and to Scala `Seq`s. It's kind of an implicit dependency already. You can read more about what I've done [here](https://tpolecat.github.io/doobie/docs/12-Custom-Mappings.html). However it's my first time contributing code to the Navigator so I'll follow your lead if you have suggestion of a different approach to take.

Reply:
Currently Navigator reads and writes lists of strings using explicit JSON serialization, see for example https://github.com/digital-asset/daml/blob/master/navigator/backend/src/main/scala/com/digitalasset/navigator/data/EventRow.scala#L38. For the sake of consistency, could you use the same approach for serializing signatories and observers?

Reply:
I am using explicit JSON serialization. The implicit `Get` and `Set` objects are for the database. You have to use those instances to make this line compile: https://github.com/digital-asset/daml/pull/1814/files#diff-837ddb57c1166f122ea0b13079991fa8R231

Or is there a different approach to take here?

Reply:
The implementation of the implicit `Get` and `Set` for the database are basically instructing `doobie` on how to read and write `Seq[String]`s, they are here: https://github.com/digital-asset/daml/pull/1814/files#diff-85802f34f8e71d1b630f3f6a7c4da9c4

Reply:
As discussed offline, I'll change this to represent JSON coming from the database as a string and parsing the string itself in the application logic.

Reply:
Addressed by https://github.com/digital-asset/daml/pull/1814/commits/f5bfecb6412d8d1d4261924ebaebf33750518d89